demo/.env

```diff
@@ -0,0 +1,5 @@
+COMPOSE_PROJECT_NAME=sciencebeam-texture
+IMAGE_TAG=develop
+UNAME=testuser
```

Comment:
`uname` is a binary that outputs system information such as the kernel version, if this is meant to be a username I would rename it in that direction